digdag-standards/build.gradle

```diff
@@ -40,7 +40,6 @@ dependencies {
     // gcs
     compile ('com.google.apis:google-api-services-storage:v1-rev20190910-1.30.3') {
         exclude group: 'com.google.guava', module: 'guava-jdk5'
-        exclude group: 'com.google.api-client', module: 'google-api-client'
```

Comment:
I have no ideas about the reason of this exclusion, but my slight concern is if this change impacts current `bq` or `gcs` related operators.

Reply:
I agree with you. But GcsWaitIT failed as #1547 and this PR will fix it.
This PR resumes integration tests for BigQuery and GcsWait. So we can check basic functionality for it.